x/stake/keeper/migrate.go

```diff
@@ -27,3 +27,21 @@ func MigratePowerRankKey(ctx sdk.Context, keeper Keeper) {
 		keeper.SetNewValidatorByPowerIndex(ctx, val)
 	}
 }
+
+func MigrateValidators(ctx sdk.Context, k Keeper) {
+	store := ctx.KVStore(k.storeKey)
+	iterator := sdk.KVStorePrefixIterator(store, ValidatorsKey)
+	defer iterator.Close()
+
+	bondDenom := k.BondDenom(ctx)
+	var err error
+	for ; iterator.Valid(); iterator.Next() {
+		validator := types.MustUnmarshalValidator(k.cdc, iterator.Value())
+		validator.DistributionAddr = types.GenerateDistributionAddr(validator.OperatorAddr, types.MockSideChainIDForBeaconChain)
+		k.SetValidator(ctx, validator)
+		_, err = k.Delegate(ctx, validator.FeeAddr, sdk.NewCoin(bondDenom, 0), validator, false)
```

Comment:
Not understand why amount is `0` here, may need to read the `node` repo..

Reply:
Tried to trigger `SetDelegation` with this. Changed to a easier way.